docs/source/generate_verification_docs.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
This script is run when build the documentation; It automatically creates `.rst` file for each verification included in the default library.